### ISSUE TITLE
As defined by the library MetaPruner, can example_inputs not be a list that contains tensor?

### ISSUE THREAD

**Issue by qyk-pc** [VainF/Torch-Pruning#460]
def __init__(
        self,
        # Basic
        model: nn.Module, # a simple pytorch model
        example_inputs: torch.tensor, # a dummy input for graph tracing. Should be on the same
        importance: typing.Callable, # tp.importance.Importance for group importance estimation
        global_pruning: bool = False, # https://pytorch.org/tutorials/intermediate/pruning_tutorial.html#global-pruning.
        pruning_ratio: float = 0.5,  # channel/dim pruning ratio, also known as pruning ratio
        pruning_ratio_dict: typing.Dict[typing.Union[nn.Module, typing.Tuple[nn.Module]], float] = None, 
        ......

**Comment by VainF** [comment]
Hi @qyk-pc, sorry for the confusing typing. The input can be a Dict.
